bpf/lib/common.h

```diff
@@ -575,6 +575,13 @@ enum metric_dir {
  */
 #define MARK_MAGIC_SNAT_DONE		0x1500
 
+/* The mark is used to indicate that the WireGuard tunnel device has done
+ * encrypting a packet. The MSB invades the Kubernetes mark "space" which is
+ * fine, as it's not used by K8s. See pkg/datapath/linux/linux_defaults/mark.go
+ * for more details.
+ */
+#define MARK_MAGIC_WG_ENCRYPTED		0x1E00
```

Comment:
I think it's already there:
```
	// MagicMarkWireGuardEncrypted is set by the WireGuard tunnel device
	// in order to indicate that a packet has been encrypted, and that there
	// is no need to forward it again to the WG tunnel netdev.
	//
	// The mark invades the K8s mark space described above. This is because
	// some packets might carry a security identity which is indicated with
	// MagicMarkIdentity which takes all 4 bits. The LSB bit which we take
	// from the K8s space is not used, so this is fine).
	MagicMarkWireGuardEncrypted int = 0x1E00
```

Reply:
Yep, added in a subsequent commit...